modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.js

```diff
@@ -116,6 +115,13 @@ export default class ScenegraphLayer extends Layer {
     this._deleteScenegraph();
   }
 
+  _applyAllAttributes(scenegraph) {
+    const allAttributes = this.getAttributeManager().getAttributes();
+    scenegraph.traverse(model => {
```

Comment:
```
traverse(node => { 
  if (node.model) {
     ...
```

Reply:
traverse is supposed to only callback on models.

Reply:
Ah yes, I recall that there are different assumptions on how `traverse` should work. 

I have always seen `traverse` as a generic mechanism for traversing arbitrary nodes in the scene "graph". 

* What if the user wants to traverse the graph to find and process camera nodes, light nodes etc? 
* What happens if/when we add `MeshNode` etc?

Maybe we can just avoid settling this now by adding the simple if guard?